configure-ironic.sh

```diff
@@ -5,11 +5,13 @@ export IRONIC_KEY_FILE=/certs/ironic/tls.key
 export IRONIC_CACERT_FILE=/certs/ca/ironic/tls.crt
 export IRONIC_INSPECTOR_CERT_FILE=/certs/ironic-inspector/tls.crt
 export IRONIC_INSPECTOR_CACERT_FILE=/certs/ca/ironic-inspector/tls.crt
+export MARIADB_CACERT_FILE=/certs/ca/mariadb/tls.crt
 
 mkdir -p /certs/ironic
 mkdir -p /certs/ironic-inspector
 mkdir -p /certs/ca/ironic
 mkdir -p /certs/ca/ironic-inspector
+mkdir -p /certs/ca/mariadb
```

Comment:
I don't think this is needed, if the CA cert file doesn't exist we aren't using this directory

Reply:
Fixed! Thanks